### PR TITLE
DMP-1192 - View Courthouse (Details Tab)

### PR DESCRIPTION
**Pull request by joshw-hmcts**
### Jira link (if applicable)

[DMP-1192](https://tools.hmcts.net/jira/browse/DMP-1192)

### Change description ###

- Stub updated to return stub users and search by name and / or email
- Stub updated to return courthouse and security-groups
- Introduces new functions into services to pick up single courthouse and security-groups
- Component created for courthouse record
- Unit tests added to other modules where coverage was low

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/app/admin/services/user-admin/user-admin.service.ts [hunk 1]
```diff
@@ -4,10 +4,10 @@ import { HttpClient } from '@angular/common/http';
 import { Injectable, inject } from '@angular/core';
 import { DateTime } from 'luxon';
 import { Observable, map } from 'rxjs';
-import { UserData } from '../models/users/user-data.interface';
-import { UserSearchFormValues } from '../models/users/user-search-form-values.type';
-import { UserSearchRequest } from '../models/users/user-search-request.interface';
-import { User } from '../models/users/user.type';
+import { UserData } from '../../models/users/user-data.interface';
```
Comment (DavidTMann) [inline src/app/admin/services/user-admin/user-admin.service.ts:7]: Can we use @ import alias's please

Comment (joshw-hmcts) [reply src/app/admin/services/user-admin/user-admin.service.ts]: Changed to use @admin-types

File: src/app/admin/components/users/users.component.ts [hunk 2]
```diff
@@ -8,8 +8,8 @@ import { Subject } from 'rxjs/internal/Subject';
 import { of } from 'rxjs/internal/observable/of';
 import { switchMap } from 'rxjs/internal/operators/switchMap';
 import { tap } from 'rxjs/internal/operators/tap';
-import { UserSearchFormValues } from '../../models/users/user-search-form-values.type';
-import { UserAdminService } from './../../services/user-admin.service';
+import { UserSearchFormValues } from '@admin-types/users/user-search-form-values.type';
+import { UserAdminService } from '../../services/user-admin/user-admin.service';
```
Comment (DavidTMann) [inline src/app/admin/components/users/users.component.ts:12]: Can use @services here

File: src/app/core/components/common/details-table/details-table.component.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { DetailsTableLink } from './../../../models/details-table/details-table-array.interface';
```
Comment (DavidTMann) [inline src/app/core/components/common/details-table/details-table.component.ts:1]: @admin-types here

**Review by DavidTMann — APPROVED** [review]

**Pull request merged**